src/lib.rs

```diff
@@ -63,6 +63,7 @@ pub(crate) mod asm;
 
 pub mod addr;
 pub mod instructions;
+pub mod ops;
```

Comment:
I placed this at the root of the crate as it looks like it's going to be applied to both addresses and sub-modules under paging. Let me know if you feel like this should live somewhere else.

Reply:
This is a great place to put this. It also makes it clear to future contributors where to put other arithmetic traits.